packages/reward-root-submitter/reward_root_submitter/config.py

```diff
@@ -0,0 +1,39 @@
+from functools import lru_cache
+
+import boto3
+from pydantic import BaseSettings, Field, root_validator
+
+
+@lru_cache
+def get_secrets_client():
+    return boto3.client("secretsmanager")
+
+
+@lru_cache
+def get_secret(secret_id):
+    client = get_secrets_client()
+    secret_value = client.get_secret_value(SecretId=secret_id)
+    return secret_value["SecretString"]
+
+
+class Config(BaseSettings):
+
+    environment: str
+    evm_full_node_url: str
+    reward_root_submitter_address: str
+    reward_root_submitter_private_key: str
+    reward_root_submitter_sentry_dsn: str
+    log_level: str = Field("WARNING")
```

Comment:
By [CS-4182](https://linear.app/cardstack/issue/CS-4182/root-submitter-no-logs), do we want to make the defaults into "INFO" or make a logging class which is more informative

Reply:
Good idea, done.